docs/warehouse/warehouse_starter_kit.md

```diff
@@ -40,10 +40,7 @@ For a given day:
     * Detailed information such as how passengers embark/disembark (ex: on a stop/at a station) onto a vehicle.
 
 * [fct_daily_schedule_feeds](https://dbt-docs.calitp.org/#!/model/model.calitp_warehouse.fct_daily_schedule_feeds)
-    * Use `gtfs_utils_v2.schedule_daily_feed_to_organization()`.
-    * Get information on an operator's schedule.
-    * How many different schedule types (schedule, vehicle positions, trip updates, and/or service alerts) does an operator produce?
-    * Is the URL to an operator's GTFS feed stable or not?
+    * Use `gtfs_utils_v2.schedule_daily_feed_to_organization()` to find regional feed type, gtfs dataset key, and feed type for an organization.
```

Comment:
Ok sorry continuing the pedantry but I don't think this function actually returns an *organization*. It returns the list of feeds with their names, but it doesn't return an organization record or a map to one.

Reply:
Yeah sorry it is a bit confusing, but nothing shown there is actually a link to an organization record or based on an organization. Everything shown is just from a combination of feeds & datasets.

Reply:
If you look at the function definition, it does not reference organizations at all: https://github.com/cal-itp/data-analyses/blob/main/_shared_utils/shared_utils/gtfs_utils_v2.py#L236-L296. If it's possible to rename the function that might help clarify, but I defer to you and @tiffanychu90 on how to handle.

Reply:
I think I get what you mean now, the `name` just references to the feed's name. Is that right?

Reply:
Yeah exactly! And again I realize that this is all kind of very particular, but the GTFS/service/organization relationship is complicated so I err on the side of over-explaining, just in case it does matter (like, sometimes these distinctions may not be that important, but other times you may really need an organization **record** and it's confusing if that's not what it is)